src/components/Patient/ManagePatients.tsx

```diff
@@ -804,169 +803,116 @@ export const PatientManager = () => {
   );
 
   return (
-    <Page
-      title={t("patients")}
-      hideBack={true}
-      breadcrumbs={false}
-      className="px-4 md:px-6"
-      options={
-        <div className="flex w-full flex-col items-center justify-between lg:flex-row">
-          <div className="mb-2 flex w-full flex-col items-center lg:mb-0 lg:w-fit lg:flex-row lg:gap-5">
-            <ButtonV2
-              id="add-patient-details"
-              onClick={() => {
-                const showAllFacilityUsers = ["DistrictAdmin", "StateAdmin"];
+    <>
+      <div className="flex w-full flex-col items-center justify-between lg:flex-row">
+        <div></div>
+        <div className="flex w-full flex-col items-center justify-end gap-2 lg:ml-3 lg:w-fit lg:flex-row lg:gap-3">
+          <Tabs
+            tabs={[
+              { text: t("live"), value: 0 },
+              { text: t("discharged"), value: 1 },
```

Comment:
I would have added it into the parent tab 
<img width="787" alt="image" src="https://github.com/user-attachments/assets/22657dbc-5613-4c4e-bcab-3211c5cc0734" />

Reply:
lets make it different routes.
/patients - Search
/patients/live - Live
patients/discharged - Discharged